refinery/core/tests.py

```diff
@@ -1492,6 +1492,52 @@ def setUp(self):
             "910117c5-fda2-4700-ae87-dc897f3a5d85"
             ]
 
+    def test_get_resources_for_user(self):
+        anon_user = AnonymousUser()
```

Comment:
I would use `get_anaoymous_user()` here to be consistent, but other than that tests look good to me.